honeybee_radiance_command/options/rpict.py

```diff
@@ -0,0 +1,963 @@
+            3,
+            float,
+        )
+        self._ma = TupleOption(
+            "ma",
+            "mist scattering albedo - default: 0.000000 0.000000 0.000000",
+            None,
+            3,
+            float,
+        )
+        self._mg = NumericOption(
+            "mg", "mist scattering eccentricity - default: 0.000000"
+        )
+        self._ms = NumericOption("ms", "mist sampling distance - default: 0.000000")
+        self._lr = IntegerOption("lr", "limit reflection - default: 7")
+        self._lw = NumericOption("lw", "limit weight - default: 1.00e-03")
+        self._am = NumericOption("am", "max photon search radius - default: 0.0")
+        self._on_setattr_check = True
+
+    def _on_setattr(self):
+        """This method executes after setting each new attribute.
+
+        Use this method to add checks that are necessary for OptionCollection. For
+        instance in rpict option collection -dj and -ps don't go together very well.
+        You can include a
+        check to ensure this is always correct.
+        """
+        assert not (self._dj > 0.0 and self._ps != -1), \
+            'It is usually wise to turn off image sampling when using direct jitter.'
+        assert not (self._ai.is_set and self._ae.is_set), \
+            'Both ai and ae are set. The program can use either an include list or ' \
+            'an exclude list, but not both.'
+        assert not (self._aI.is_set and self._aE.is_set), \
+            'Both aI and aE are set. The program can use either an include list or ' \
+            'an exclude list, but not both.'
+        assert (self._i and not self._dv), \
+            'If irradiance values are requested, it is better to keep -dv off so that' \
+            'light sources do not appear with their original radiance values.'
+
+    @property
+    def vt(self):
+        """view type perspective - default: vtv
+
+        1. 'v' sets a perspective view.
+        2. 'l' sets parallel view.
+        3. 'c' sets a cylindrical panaroma. This view is like a standard perspective
+            vertically, but projected on a cylinder horizontally, like a soupcan's
+            eye-view.
+        4. 'h' sets a hemispherical fisheye view. This is a projection of the hemisphere
+            onto a circle. The maximum view angle for this type is 180 degrees.
+        5. 'a' sets an angular fisheye view. An angular fisheye view is defined such
+            that distance from the center of the image is proportional to the angle
+            from the central view direction. An angular fisheye can display a full 360
+            degrees.
+        6. 's' sets a planisphere (stereographic) view. A planisphere fisheye view
+            maintains angular relationships between lines, and is commonly used for
+            sun path analysis. This is more commonly known as a stereographic projection.
+
+        Note:
+
+        There is no space between the view type option and its single letter argument.
```

Comment:
I think we should keep it because as far as I have seen, in other radiance parameters there's usually a space between a radiance parameter name and its value. For example, -ab 2

Reply:
Yes but that makes sense there because you write `-ab 2` and `-vtv` but here for a user it looks like this:

```python
options.ab = 2
options.vt = 'v'
```

And we take care of creating `-ab 2 -vtv`. The user shouldn't be concerned about the space at all. That's why I think this comment will become a source of confusion.